### PR TITLE
Mixed reality capture documentation updates

### PR DESCRIPTION
**Pull request by lordpi**
This makes a number of updates to mixed reality capture:
- Add new audio effect properties
- Update the default value of PreferredHologramPerspective
- Add information about Render from PV Camera support in Unity and Unreal
- Update image and document settings that can be configured in Windows Device Portal
- Update HoloLens Camera in Unreal terminology to better align with Mixed Reality Capture for Developers terminology

### CODE REVIEW

File: mixed-reality-docs/device-portal-api-reference.md [hunk 1]
```diff
@@ -317,7 +317,7 @@ Sets the default mixed reality capture settings.  Some of these settings are app
 
 **/api/holographic/mrc/status (GET)**
 
-Gets the status of the mixed reality recorded (running, stopped)
+Gets the state of mixed reality capture within the Windows Device Portal (IsRecoding is true while Windows Device Portal is recording video)
```
Comment (hferrone) [inline mixed-reality-docs/device-portal-api-reference.md:320]: - IsRecoding? Spelling error -> IsRecording?
- Any code variables or references should use the <code> markdown -> `IsRecording`
- Might want to mention where the user can find the IsRecording variable

File: mixed-reality-docs/mixed-reality-capture-for-developers.md [hunk 2]
```diff
@@ -78,10 +78,31 @@ When mixed reality capture stops (or if the app disables the view configuration
 
 A [ViewConfiguration](https://docs.microsoft.com/uwp/api/windows.graphics.holographic.holographiccamera.viewconfiguration) property has been added to HolographicCamera to help identify the configuration a camera belongs to.
 
+##### Enable the PhotoVideoCamera HolographicViewConfiguration in Unity
+
+To opt-in, when using the [Mixed Reality Toolkit](https://microsoft.github.io/MixedRealityToolkit-Unity/README.html), enable the [Windows Mixed Reality Camera Settings](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/CameraSystem/WindowsMixedRealityCameraSettings.html) provider and check the "Render from PV Camera" setting.
```
Comment (hferrone) [inline mixed-reality-docs/mixed-reality-capture-for-developers.md:83]: Render from PV Camera should be bold

Comment (hferrone) [inline mixed-reality-docs/mixed-reality-capture-for-developers.md:83]: Delete "To opt-in"

File: mixed-reality-docs/mixed-reality-capture-for-developers.md [hunk 3]
```diff
@@ -78,10 +78,31 @@ When mixed reality capture stops (or if the app disables the view configuration
 
 A [ViewConfiguration](https://docs.microsoft.com/uwp/api/windows.graphics.holographic.holographiccamera.viewconfiguration) property has been added to HolographicCamera to help identify the configuration a camera belongs to.
 
+##### Enable the PhotoVideoCamera HolographicViewConfiguration in Unity
+
+To opt-in, when using the [Mixed Reality Toolkit](https://microsoft.github.io/MixedRealityToolkit-Unity/README.html), enable the [Windows Mixed Reality Camera Settings](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/CameraSystem/WindowsMixedRealityCameraSettings.html) provider and check the "Render from PV Camera" setting.
+
+If not using the Mixed Reality Toolkit, you can use a component to [manually opt-in](#enable-the-photovideocamera-holographicviewconfiguration-in-directx) as described above for DirectX.
```
Comment (hferrone) [inline mixed-reality-docs/mixed-reality-capture-for-developers.md:85]: "If you're not using..."

File: mixed-reality-docs/mixed-reality-capture-for-developers.md [hunk 4]
```diff
@@ -78,10 +78,31 @@ When mixed reality capture stops (or if the app disables the view configuration
...
 A [ViewConfiguration](https://docs.microsoft.com/uwp/api/windows.graphics.holographic.holographiccamera.viewconfiguration) property has been added to HolographicCamera to help identify the configuration a camera belongs to.
 
+##### Enable the PhotoVideoCamera HolographicViewConfiguration in Unity
+
+To opt-in, when using the [Mixed Reality Toolkit](https://microsoft.github.io/MixedRealityToolkit-Unity/README.html), enable the [Windows Mixed Reality Camera Settings](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/CameraSystem/WindowsMixedRealityCameraSettings.html) provider and check the "Render from PV Camera" setting.
+
+If not using the Mixed Reality Toolkit, you can use a component to [manually opt-in](#enable-the-photovideocamera-holographicviewconfiguration-in-directx) as described above for DirectX.
+
+> [!NOTE]
+> This requires Unity 2018.4.13f1 or newer or 2019.3.0f1 or newer.
```
Comment (hferrone) [inline mixed-reality-docs/mixed-reality-capture-for-developers.md:88]: Version note doesn't make sense. If it requires 2019.3.0f1 or newer, that would cover 2018.4.13f1?

File: mixed-reality-docs/mixed-reality-capture-for-developers.md [hunk 5]
```diff
@@ -78,10 +78,31 @@ When mixed reality capture stops (or if the app disables the view configuration
...
+
 ##### Handle the additional HolographicCamera render in Unity
 
->[!NOTE]
-> Unity support to render from the PV camera is under development and can't be used, yet. This documentation will be updated when a build of Unity with support for rendering from the PV camera is available.
+This is done for you automatically by Unity.
+
+##### Enable the PhotoVideoCamera HolographicViewConfiguration in Unreal
+
+To opt-in, call SetEnabledMixedRealityCamera and ResizeMixedRealityCamera with the desired video dimensions.
```
Comment (hferrone) [inline mixed-reality-docs/mixed-reality-capture-for-developers.md:96]: SetEnabledMixedRealityCamera and ResizeMixedRealityCamera should use the <code> markdown tag.

File: mixed-reality-docs/unreal-hololens-camera.md [hunk 6]
```diff
@@ -1,26 +1,36 @@
...
 
-Third camera Mixed Reality Capture (MRC) can be used to render a mixed reality capture from the perspective of the camera on the HoloLens visor, rather than the perspective of the eye textures.  This improves the mapping between the real world and the holograms in the MRC video. 
+## Rendering from the PV Camera for MRC
 
-To opt into using third camera MRC, call SetEnabledMixedRealityCamera and ResizeMixedRealityCamera with the desired video dimensions. 
+The system, and custom MRC recorders, have the ability to create MRC photos, videos, and streams.
+
+For better hologram alignment, the app can choose to render from the perspective of the PV Camera. Otherwise, an eye texture is used.
+
+To opt-in, call SetEnabledMixedRealityCamera and ResizeMixedRealityCamera with the desired video dimensions.
```
Comment (hferrone) [inline mixed-reality-docs/unreal-hololens-camera.md:21]: SetEnabledMixedRealityCamera and ResizeMixedRealityCamera should use the code markdown tag.

File: mixed-reality-docs/device-portal-api-reference.md (hunk 1 above)
Comment (lordpi) [reply mixed-reality-docs/device-portal-api-reference.md:320]: Good call on the spelling error.  It doesn't look like anything is documenting its return values well, but I'll do so

File: mixed-reality-docs/mixed-reality-capture-for-developers.md (hunk 4 above)
Comment (lordpi) [reply mixed-reality-docs/mixed-reality-capture-for-developers.md:88]: Unity major versions are separately maintained/supported.  The two builds shipped at about the same time, targeting different Unity major versions.

**Comment by lordpi** [comment]
@hferrone I ran into some merge conflicts in unreal-hololens-camera.md, since it seems we both made similar edits. I changed it to be somewhat of a combination of our two edits, so let me know what you think.

**Comment by hferrone** [comment]
@lordpi - combination looks fine to me, thanks for including the changes!

**Comment by mattzmsft** [comment]
@lordpi @jessemcculloch @hferrone I LOVE these updates. Thanks so much. I noticed one issue, however, in the updates to the "Using the Windows Device Portal" page: in the Mixed Reality Capture section, under the bullets describing what each feature does, it looks like the table's formatting is messed up. It's showing the brackets and dashes of the markdig instead of formatting as a table.

**Comment by mattzmsft** [comment]
I think I fixed it. Let me confirm :)

**Comment by mattzmsft** [comment]
Okay, looks fixed now on review.docs, so it should be good on the public page the next time you push to live!

**Pull request merged**